pandas/core/series.py

```diff
@@ -374,6 +374,10 @@ def get_values(self):
         """ same as values (but handles sparseness conversions); is a view """
         return self._data.get_values()
 
+    @property
```

Comment:
actually maybe could define this in ``core/base/IndexOpsMixin`` (as ``Index.asobject``) already exists